### PR TITLE
feat(config): rename config properties

### LINKED ISSUE #160: Change the caching/aggregation to be generic to messages
Currently the Postee caching and aggregation mechanisms are specific to vulnerability messages.
We would like them to become generic to every input message.

To achieve this we need to do the following changes:

== cfg.yaml ==
1. policy-show-all
Change to "unique-message-fields" which is an array of message fields that are used to identify if message was already handled. 

2. Add a new "unique-message-timeout" parameter, which controls for how much time a message is cached (1h, 1d, 1w). If empty then cached forever
 
3. aggregate-issues-number
Should be changed to "aggregate-message-number". We will aggregate messages in a route (not just "issues").

4. aggregate-issues-timeout
Should be changed to "aggregate-message-timeout".

### PR DESCRIPTION
**Pull request by AndreyLevchenko**
Closes #160

### CODE REVIEW

File: msgservice/calculateexpired_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			1,
+		},
+		{
+			2,
+		},
+		{
+			100,
+		},
+	}
+
```
Comment (afdesk) [inline msgservice/calculateexpired_test.go:27]: this struct looks like an int array.

File: routes/aggrtimeout.go [hunk 2]
```diff
@@ -6,34 +6,50 @@ import (
 	"strings"
 )
 
-func ConfigureAggrTimeout(route *InputRoute) *InputRoute {
+func parseTimeouts(v string) (int, error) {
```
Comment (afdesk) [inline routes/aggrtimeout.go:9]: `parseTimeouts` returns an error for an empty input `v`.
Error: `strconv.Atoi: parsing "": invalid syntax`

File: routes/aggrtimeout.go [hunk 3]
```diff
@@ -6,34 +6,50 @@ import (
...
+
+func ConfigureTimeouts(route *InputRoute) *InputRoute {
+	if len(route.Plugins.AggregateMessageTimeout) > 0 {
+		aggregateTimeoutSeconds, err := parseTimeouts(route.Plugins.AggregateMessageTimeout)
+		if err != nil {
+			log.Printf("%q settings: Can't convert 'aggregate-message-timeout'(%q) to seconds.",
+				route.Name, route.Plugins.AggregateMessageTimeout)
 		}
+
+		route.Plugins.AggregateTimeoutSeconds = aggregateTimeoutSeconds
```
Comment (afdesk) [inline routes/aggrtimeout.go:43]: is this string really need?
```go
	var err error
	route.Plugins.UniqueMessageTimeoutSeconds, err = parseTimeouts(route.Plugins.AggregateMessageTimeout)
	if err != nil {
		log.Printf("%q settings: Can't convert 'aggregate-message-timeout'(%q) to seconds.",
			route.Name, route.Plugins.AggregateMessageTimeout)
	}
```

File: routes/aggrtimeout.go (hunk 2 above)
Comment (afdesk) [reply routes/aggrtimeout.go:9]: also it won't parse strings with a spaces `1 m`.

**Review by afdesk — CHANGES_REQUESTED** [review]

File: routes/aggrtimeout.go (hunk 3 above)
Comment (AndreyLevchenko) [reply routes/aggrtimeout.go:43]: I've got an compilation error with code you suggested

File: routes/aggrtimeout.go (hunk 2 above)
Comment (AndreyLevchenko) [reply routes/aggrtimeout.go:9]: fixed

File: msgservice/calculateexpired_test.go (hunk 1 above)
Comment (AndreyLevchenko) [reply msgservice/calculateexpired_test.go]: fixed

File: routes/aggrtimeout.go (hunk 3 above)
Comment (afdesk) [reply routes/aggrtimeout.go]: it works for me. checked.
```go
		var err error
		route.Plugins.AggregateTimeoutSeconds, err = parseTimeouts(route.Plugins.AggregateMessageTimeout)
		if err != nil {
			log.Printf("%q settings: Can't convert 'aggregate-message-timeout'(%q) to seconds.",
				route.Name, route.Plugins.AggregateMessageTimeout)
		}
```

Comment (afdesk) [reply routes/aggrtimeout.go]: it doesn't matter.

**Review by afdesk — APPROVED** [review]

**Pull request merged**